fleece/cli/build/build.py

```diff
@@ -323,10 +331,13 @@ def _build(service_name, python_version, src_dir, requirements_path,
         detach=True)
     for line in container.logs(stream=True, follow=True):
         sys.stdout.write(line.decode('utf-8'))
-    exit_code = container.wait()
+    status = container.wait()
+    exit_code = status.get('StatusCode')
```

Comment:
Will there be any issues if `StatusCode` is None?

Reply:
hmm, actually - we should probably error out if StatusCode is also None since something went wrong. The [docs](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.wait) just say that `wait` returns a `dict` - so...hard to say what's guaranteed to be in it.